tests/python/test_sklearn_integration.py

```diff
@@ -192,7 +192,7 @@ def test_skl_converter_iforest(dataset):
 
     tl_model = treelite.sklearn.import_model(clf)
     out_pred = treelite.gtil.predict(tl_model, X)
-    np.testing.assert_almost_equal(out_pred, expected_pred, decimal=2)
```

Comment:
Previously, this test case was using a loose error bound (`decimal=2`). With the fix, we are now able to use a tighter error bound.